src/beta_inc.jl

```diff
@@ -825,3 +825,120 @@ function beta_inc(a::T, b::T, x::T, y::T) where {T<:Union{Float16, Float32}}
 end
 beta_inc(a::Real, b::Real, x::Real, y::Real) = beta_inc(promote(float(a), float(b), float(x), float(y))...)
 beta_inc(a::T, b::T, x::T, y::T) where {T<:AbstractFloat} = throw(MethodError(beta_inc,(a, b, x, y,"")))
+
+#GW Cran, KJ Martin, GE Thomas, Remark AS R19 and Algorithm AS 109: A Remark on Algorithms AS 63: The Incomplete Beta Integral and AS 64: Inverse of the Incomplete Beta Integeral,
+#Applied Statistics,
+#Volume 26, Number 1, 1977, pages 111-114. 
+
+"""
+    beta_inc_inv(a,b,p,q,lb=logbeta(a,b)[1])
+
+Computes inverse of incomplete beta function. Given `a`,`b` and ``I_{x}(a,b) = p`` find `x` and return tuple (x,y).
+"""
+function beta_inc_inv(a::Float64, b::Float64, p::Float64, q::Float64; lb = logbeta(a,b)[1])
+    fpu = 1e-30
+    x = p
+    if p == 0.0
+        return (0.0, 1.0)
+    elseif p == 1.0
+        return (1.0, 0.0)
+    end
+
+    #change tail if necessary
+
+    if p > 0.5
+        aa = q
+        pp = b
+        qq = a
+        indx = true
+    else
+        aa = p
+        pp = a
+        qq = b
+        indx = false
```

Comment:
lets try to keep all the symbols consistent:
`aa` should be `pp`
`pp` should be `aa`
`qq` should be `bb`